src/content/en/2021/media.md

```diff
@@ -1,20 +1,900 @@
 ---
 #See https://github.com/HTTPArchive/almanac.httparchive.org/wiki/Authors'-Guide#metadata-to-add-at-the-top-of-your-chapters
 title: Media
-description: TODO
+description: Media chapter of the 2021 Web Almanac covering how images and videos are currently being encoded, embedded, styled, and delivered on the web
 authors: [eeeps, dougsillars]
-reviewers: []
-analysts: [eeeps, dougsillars]
+reviewers: [Navaneeth-akam, tpiros, akshay-ranganath, addyosmani]
+analysts: [eeeps, dougsillars, akshay-ranganath]
 editors: []
 translators: []
+eeeps_bio: Eric Portis is a Web Platform Advocate at <a hreflang="en" href="https://cloudinary.com/">Cloudinary</a>.
+dougsillars_bio: Doug Sillars is a freelance digital nomad working on the intersection of performance and media. He tweets [@dougsillars](https://twitter.com/dougsillars), and blogs regularly at <a hreflang="en" href="https://dougsillars.com">dougsillars.com</a>.
 results: https://docs.google.com/spreadsheets/d/1nwkpviC3gNhRb48i8OoIgtJx1ckqRjJGW7uc7Gdi_sI/
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
+featured_quote: This year, with native lazyloading, we collectively stemmed the tide of ever-increasing image transfer sizes.
+featured_stat_1: 99.93%
+featured_stat_label_1: Pages that load an image resource of some kind.
+featured_stat_2: 87.1%
+featured_stat_label_2: Images on the web delivered as JPEGs, PNGs, or GIFs insteasd of next-gen formats!
+featured_stat_3: 5.6%
+featured_stat_label_3: Share of desktop pages that included a `video` element. A three-fold increase from last year.
+unedited: true
 ---
 
-## TODO
+## Introduction
+
+Almost three decades ago <a hreflang="en" href="https://thehistoryoftheweb.com/the-origin-of-the-img-tag/">the `<img>` tag dropped</a> and hyper_text_ became hyper_media_. The web has become increasingly visual, ever since. What is the state of media on the web in 2021? Let's look at images and videos, in turn.
+
+## Images
+
+Images are ubiquitous on the web.
+
+{{ figure_markup(
+  content="96%",
+  caption="Percentage of pages that contained at least one contentful `<img>`",
+  classes="big-number",
+  sheets_gid="1756671383",
+  sql_file="at_least_one_img.sql"
+)
+}}
+
+{{ figure_markup(
+  content="99.93%",
+  caption="Percentage of pages that generated at least one request for an image resource",
+  classes="big-number",
+  sheets_gid="1062090109",
+  sql_file="at_least_one_image_request.sql"
+)
+}}
+
+Put simply, almost every page contains image content and effectively all pages serve up _some_ sort of imagery (even if it's just a background or favicon).
```

Comment:
We've two big figures one after the other, which I don't think looks the best. Should we lose one?

Reply:
FYI you can see what it looks like here: https://20211207t160821-dot-webalmanac.uk.r.appspot.com/en/2021/media#images

Reply:
You have a point. Perhaps I'll move a sentence between them to break them up.